pkg/namesgenerator/names-generator.go

```diff
@@ -149,6 +165,12 @@ var (
 		// Leonardo Da Vinci invented too many things to list here. https://en.wikipedia.org/wiki/Leonardo_da_Vinci.
 		"davinci",
 
+		// Donna Dubinsky, CEO and co-founder of Palm, Inc. https://en.wikipedia.org/wiki/Donna_Dubinsky
+		"dubinsky",
```

Comment:
Will probably not make me popular; is co-founding a company notable enough to make it in this list?

(Read the linked Wikipedia page; there are some notable things in there, so possibly with a better description)